raiden/transfer/mediated_transfer/transition.py

```diff
@@ -12,10 +12,10 @@ def update_route(next_state, route_state_change):
 
     # TODO: what if the route that changed is the current route?
 
-    if new_route.state == 'unavailable':
+    if new_route.state != 'opened':
         available_routes.pop(available_idx)
 
-    elif new_route.state == 'available':
+    elif new_route.state == 'opened':
```

Comment:
Since we're comparing `Channel` states, I would suggest using class fields in the `Channel` class:
```
class Channel(object):
  STATE_OPENED = 'opened'
  STATE_CLOSED = 'closed'
  ...
```

however, they wouldn't be immutable, so there is still chance for fuckup, if someone ends up assigning new values to them.